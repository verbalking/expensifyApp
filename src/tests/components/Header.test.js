import {shallow} from 'enzyme'
import React from 'react';
import Header from '../../components/Header';

test ('shuld render Header', ()=>{
  const wrapper = shallow(<Header/>);
  expect(wrapper).toMatchSnapshot();
  // const renderer = new ReactShallowRenderer();
  // renderer.render(<Header/>);
  // expect(renderer.getRenderOutput()).toMatchSnapshot();
  // expect(wrapper.find('h1').text()).toBe('Expensify');
});

