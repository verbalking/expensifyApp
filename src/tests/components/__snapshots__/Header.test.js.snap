// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shuld render Header 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <[object Object]
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    Create page
  </[object Object]>
</header>
`;
